2.1.0/scratch/get-scratch-bins.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+# get-scratch-bins.sh builds different architecture NATS server binaries from
+# source inside of a Go container. These binaries are then exposed to the host
+# and checked into this repo.
+#
+# The reason we do this and not use the official release binaries is so we can
+# disable CGO. We have to disable CGO in order to run NATS inside of a scratch
+# container.
+#
+# Note 'EOF' means that the heredoc is sent as a string literal. No variable
+# expansion happens from the super shell.
+
+docker run --interactive --rm --volume $(pwd):/out golang:1.12.9 bash << 'EOF'
+	set -ex
+	mkdir -p /go/src/github.com/nats-io/nats-server
+	cd /go/src/github.com/nats-io/nats-server
+
+	# Version done this way so it's easy to search and replace. This is the
+	# format that the Dockerfiles have for specifying the version.
+	ver=( NATS_SERVER 2.1.0 )
+	git clone --branch "v${ver[1]}" --depth 1 https://github.com/nats-io/nats-server.git .
+
+	export CGO_ENABLED=0
+	export GO111MODULE=off
+
+	commit="$(git rev-parse --short HEAD)"
+
+	GOARCH=amd64 go build -o /out/amd64/nats-server -v -a -tags netgo -installsuffix netgo \
```

Comment:
Images are built and put into `/out/..` but that's not where the server exes are located. They are in 2.1.0/scratch/<arch>?

Reply:
`get-scratch-bins.sh` is meant to be run in the `scratch` directory. The Go container is started with

```
--volume $(pwd):/out
```

which means that the current directory, which should contain the different architecture folders, will be mounted as `/out` in the container.

Then, in the container binaries get written to `/out`. So the mapping is like this.

```
host       container
./amd64 -> /out/amd64
./arm64v8 -> /out/arm64v8
```

I added a check to the script to make sure it doesn't get run from the top level of the repo.

Reply:
I missed that. Thanks!